public/static/docs/command-reference/status.md

```diff
@@ -19,25 +19,27 @@ positional arguments:
 ## Description
 
 `dvc status` searches for changes in the existing pipelines, either showing
-which [stages](/doc/command-reference/run) have changed in the workspace and
-must be reproduced (with `dvc repro`), or differences between cache vs. remote
-storage (meaning `dvc push` or `dvc pull` should be run to synchronize them).
-The two modes, _local_ and _cloud_ are triggered by using the `--cloud` or
-`--remote` options:
+which [stages](/doc/command-reference/run) have changed in the workspace
+(including uncommitted local changes) and must be reproduced (with `dvc repro`),
+or differences between <abbr>cache</abbr> vs. remote storage (meaning `dvc push`
+or `dvc pull` should be run to synchronize them). The two modes, _local_ and
+_cloud_ are triggered by using the `--cloud` or `--remote` options:
 
 | Mode   | CLI Option | Description                                                                                                                 |
 | ------ | ---------- | --------------------------------------------------------------------------------------------------------------------------- |
 | local  | _none_     | Comparisons are made between data files in the workspace and corresponding files in the cache directory (e.g. `.dvc/cache`) |
 | remote | `--remote` | Comparisons are made between the cache, and the given remote. Remote storage is defined using the `dvc remote` command.     |
-| remote | `--cloud`  | Comparisons are made between the cache, and the default remote, defined with `dvc remote --default` command.                |
+| remote | `--cloud`  | Comparisons are made between the cache, and the default remote, typically defined with `dvc remote --default`.              |
 
-DVC determines data and code files to compare by analyzing all
-[DVC-files](/doc/user-guide/dvc-file-format) in the <abbr>project</abbr>
-(`--all-branches` and `--all-tags` in the `cloud` mode compare multiple
-workspace versions). The comparison can be limited to specific DVC-files by
-listing them as `targets`. Changes are reported only against the given
-`targets`. When combined with the `--with-deps` option, a search is made for
-changes in other stages that affect the target.
+DVC determines which data and code files to compare by analyzing all
+[DVC-files](/doc/user-guide/dvc-file-format) in the <abbr>workspace</abbr>
+(`--all-branches` and `--all-tags` use multiple <abbr>repository</abbr>
+[revisions](https://git-scm.com/docs/revisions)).
```

Comment:
this sounds better (revisions and stuff), but I really believe that the previous one with workspace versions (even though not technically correct) was way easier to understand

Reply:
- I think "`--all-branches` and `--all-tags` in the `cloud` mode" was not easy to understand so I removed "in the cloud mode"
- I agree "`--all-branches` and `--all-tags` compare multiple workspace versions" is better than "... use multiple repository revisions" though, restored now.